### ISSUE TITLE
Classifying Meanings & Documents (E2) - So & Long 2015

### ISSUE THREAD

**Issue by HyunkuKwon** [UChicago-CCA-2021/Readings-Responses#15]
Post questions here about the following exemplary reading:

So, Richard and Hoyt Long. 2015. “[Literary Pattern Recognition: Modernism between Close Reading and Machine Learning.](https://www.journals.uchicago.edu/doi/abs/10.1086/684353?journalCode=ci)” Critical Inquiry 42(2): 235-267.

**Comment by yushiouwillylin** [comment]
I want to ask a question to make sure I understood this paper correctly. It starts from aiming to propose a new model for reading literary combining computation method and humanistic approach. The method seems to start from using classification algorithm, then use close reading or sociohistorical approaches to classify literature types. In the end researchers look closely into the vague cases, either except there's some higher statistical pattern they did not capture, or just forget about it. 

It seems like this is a comparison across methods, not really a new model that combines computational method and humanistic approaches, is this a valid criticism? Or did I lose some important details along the way?

**Comment by Bin-ary-Li** [comment]
I applaud the authors' enthusiasm about using machine learning techniques to analyze literary texts, specifically English haiku. But when I started at the beginning of the paper I thought I was promised a full explication about the epistemology of the statistical property of literary objects. I was waiting for a grand philosophical battle about the essence of knowledge and representation. Instead, I felt like I was reading an NLP paper in disguise. I don't think those critics on the other side (e.g. Alexander Galloway) will be proselytized by an experiment report that employs a methodology they fundamentally disagree with. But what does it take to really bridge the gap?

**Comment by chiayunc** [comment]
I agree very much with @Bin-ary-Li . The title suggests that it is going to tackle the issue from somewhere closer to the core of the debate the authors introduced in their introduction. In the end, they used Haiku as the main subject corpus of the study; the study quite frankly relied on the rhythmic, therefore statistical as property associated with the literature. This is probably more like a comment, but I think some kind of epistemology approach that gets under the hood of the interaction of humanistic understanding and machine learning required for literary pattern recognition would be highly-coveted.

**Comment by egemenpamukcu** [comment]
Haiku seems to have very unique characteristics that distinguishes it from other forms of literary texts, I was wondering whether such classification methods would generalize into other forms of less clear-cut literary texts without losing its precision.
